operator/config/manager/manager.yaml

```diff
@@ -25,32 +25,35 @@ spec:
       securityContext:
         runAsNonRoot: true
       containers:
-      - command:
-        - /manager
-        args:
-        - --leader-elect
-        image: controller:latest
-        name: manager
-        securityContext:
-          allowPrivilegeEscalation: false
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 15
-          periodSeconds: 20
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 8081
-          initialDelaySeconds: 5
-          periodSeconds: 10
-        resources:
-          limits:
-            cpu: 100m
-            memory: 30Mi
-          requests:
-            cpu: 100m
-            memory: 20Mi
+        - args: ["proxy", "--port=8231"]
+          image: bitnami/kubectl:latest
```

Comment:
Here I am using [bitnami's image](https://hub.docker.com/r/bitnami/kubectl/) which has more than 1 billion pulls and the build files are open sources https://github.com/bitnami/bitnami-docker-kubectl and they seem to using a bot to automatically update the images when there is a new version of kubectl. Is this ok or should I build another custom image?

Reply:
I would rather we download kubectl into our image, that way airgap users only have to download 1 image.  We can leave this for the work next sprint.